apsuite/loco/main.py

```diff
@@ -16,8 +16,8 @@ class LOCO:
     DEFAULT_TOL = 1e-3
```

Comment:
these constants have units? if they so, I think we should include a comment in the line with it. I always use the format
```python

FREQ_DELTA = 10.0  # [Hz]

Reply:
@murilobalves , I don't know why but when I make a comment on a code line, the line that shows ups in this conversation is not necessarily the correct one. I must be misusing it.

I was looking at `DEFAULT_DELTAK_NORMALIZATION`... but please check all constants.

Reply:
Actually `DEFAULT_DELTAK_NORMALIZATION` is a parameter I would like to discuss with the group. I guess its unit will also be a result from this discussion. I just set it to 1.0 temporarily. The other class constants in LOCO are dimensionless. The constants in LOCOConfig has a comment indicating its units as you suggested.

Reply:
all right, let's discuss in gitter... aproving this PR.